programmers/Coding_Test_High_Score_Kit/farthest_node/LHS.py

```diff
@@ -0,0 +1,74 @@
+
+def solution(n, edge):
+    graph = {}
+    
+    for i in range(len(edge)):
+        a = edge[i][0]
+        b = edge[i][1]
+        graph[a] = graph.get(a, []) + [b]
+        graph[b] = graph.get(b, []) + [a]
+    # print(graph)
+
+    visit = []   
+    queue = []
```

Comment:
How about using `collections.deque` next time? It takes less time to look up an element compared to Python's built-in list.